CONTRIBUTING.md

```diff
@@ -53,7 +53,7 @@ and instructions for signing it [can be found here](https://git.k8s.io/community
 If you're new to the project and want to help, but don't know where to start, we have a semi-curated list of issues that
 should not need deep knowledge of the system. [Have a look and see if anything sounds
 interesting](https://github.com/kubernetes-sigs/cluster-api/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
-Before starting to work on the issue, make sure that it doesn't have a [lifecycle/active](https://github.com/kubernetes-sigs/cluster-api/labels/lifecycle%2Factive) label. If the issue has been assigned, reach out to the assignee.
+Before starting to work on the issue, make sure that it doesn't have a [lifecycle/active](https://github.com/kubernetes-sigs/cluster-api/issues?q=is%3Aissue%20state%3Aopen%20label%3Alifecycle%2Factive) label. If the issue has been assigned, reach out to the assignee.
```

Comment:
I think the idea was to link to the label, not to a list of issues that have the label. Let's just drop the link and use `lifecycle/active` instead